### PR TITLE
Invitation ready

### PR DESCRIPTION
**Pull request by kaihendry**
Fixes https://github.com/unee-t/frontend/issues/110

Tried to send a test mail, but it failed with sandbox credentials. Why not use the SMTP server in .env?

### CODE REVIEW

**Comment by nbiton** [comment]
It is using the SMTP server that's set in .env... Or do you mean that we should set another SMTP server url to it?

**Comment by kaihendry** [comment]
Ok, it's ready to merge imo. Can you please check it over @nbiton ?

File: imports/api/rest/put-pending-invitations.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    const results = PendingInvitations.update({
+      _id: { $in: req.body },
+      done: { $ne: true }
+    },
+      { $set: { done: true } })
+    if (results < 1) {
+      // Nothing happened, so don't send any emails
+      res.send(200, results)
+      return
+    }
```
Comment (nbiton) [inline imports/api/rest/put-pending-invitations.js]: Can you update Meteor.users for the user's invitedToCases attribute (with the right sub-document) with "done: true"?

Comment (kaihendry) [reply imports/api/rest/put-pending-invitations.js]: Btw why doesn't invitedToCases reference who invited the user? Shouldn't it?

Comment (kaihendry) [reply imports/api/rest/put-pending-invitations.js]: Need to answer the question: "How many ppl has this user invited in the last week?" for https://github.com/unee-t/frontend/issues/116

**Pull request merged**